src/main/java/duke/DukeExcpetion.java

```diff
@@ -0,0 +1,14 @@
+package duke;
+
+/**
+ * Exception when users type invalid input.
+ */
+public class DukeExcpetion extends Exception {
```

Comment:
This is something minor but Is it better if the Class is named DukeException? I think it will increase the readability of the code!